config/application.php

```diff
@@ -0,0 +1,16 @@
+<?php
+require __DIR__ . '/constants.config.php';
+
+chdir(APPLICATION_PATH);
```

Comment:
I don't really like this. I wonder if we could move away from, or if it's absolutely necessary, keep it in index.php?

Reply:
Yes, sorry for being too vague. I'm ok with keeping it, but `config/application.php` feels like wrong place for it. I'd rather keep it in index.php.

Reply:
I think it all comes down to what purpose it serves in the first place -- and honestly, I'm not sure; I think it may come from original Zend Framework boilerplate, though I could be mistaken.

If we move it to index.php, we're changing the order of operations, and if the Application initialization depends on it in some way, then we're performing the action too late.

I suppose I could just delete it and run tests and see what happens -- but I'm just concerned there may be some subtle or obscure reason for it that will bite us later.

Reply:
Okay, let's keep it for now and revisit in another PR or something. My worry is that this may bite us at some point e.g. if a utility is started from another directory with a relative path to a file as parameter.

Reply:
Sounds good. As soon as this PR gets merged, I'll open a new one to remove the chdir entirely, run tests, and report on the results. If we find a compelling reason to keep it, we can move it to public/index.php if feasible. But hopefully it can just go away and fix the relative file problem you mention.